distributed/utils_test.py

```diff
@@ -886,10 +886,11 @@ async def coro():
                                 )
                             except Exception as e:
                                 logger.error(
-                                    "Failed to start gen_cluster, retrying",
+                                    "Failed to start gen_cluster: "
+                                    f"{e.__class__.__name__}: {e}, retrying",
                                     exc_info=True,
                                 )
-                                await asyncio.sleep(1)
+                                await asyncio.sleep(5)
```

Comment:
Personally I would rather have more short sleeps than long ones.  This change may have been necessary for some reason though.

Reply:
I was systematically getting failures here. I'll try dropping down the sleep again and increasing the number of retries to compensate.